OpenEdXMobile/res/values/iap_strings.xml

```diff
@@ -52,4 +52,9 @@
     <string name="label_refresh_now">Refresh now</string>
     <!-- Label Continue Without Update -->
     <string name="label_continue_without_update">Continue without update</string>
+
+    <!-- Title of the dialog to verify SDN -->
```

Comment:
Can we add the SDN complete form in the comments for clearance? SDN stands for Specially Designated Nationals.

Reply:
IMO not about the complete form, so we can leave it for now.